lib/paper_trail/config.rb

```diff
@@ -40,7 +40,8 @@ def track_associations
 
     # Indicates whether PaperTrail is on or off.
     def enabled
-      PaperTrail.paper_trail_store[:paper_trail_enabled].nil? || PaperTrail.paper_trail_store[:paper_trail_enabled]
+      PaperTrail.paper_trail_store[:paper_trail_enabled].nil? ||
+        PaperTrail.paper_trail_store[:paper_trail_enabled]
```

Comment:
Re: second argument, that is how `Hash#fetch` works but it appears that `RequestStore.fetch` is different.

- Implementation: https://github.com/steveklabnik/request_store/blob/160c635efbc16d9c16063e34f9d15f6673299c18/lib/request_store.rb#L34
- Example: https://github.com/steveklabnik/request_store/blob/160c635efbc16d9c16063e34f9d15f6673299c18/test/request_store_test.rb#L44

So, I guess we'd do something like:

```ruby
PaperTrail.paper_trail_store.fetch(:paper_trail_enabled) { true }
```

Does that sound right?